server/app/assets/javascripts/admin_applications.ts

```diff
@@ -68,12 +69,23 @@ class AdminApplications {
       ),
     )
 
+    this.cards.forEach((card) =>
+      this._assertNotNull(card.children[0], 'card inner div').classList.add(
+        AdminApplications.BACKGROUND_WHITE_CLASS,
```

Comment:
Let's add it in function on line 67:

```
const child = this._assertNotNull(card.children[0], 'card inner div');
child.classList.remove(AdminApplications.BACKGROUND_GRAY_CLASS);
child.classList.add(AdminApplications.BACKGROUND_WHITE_CLASS);
```

Reply:
done

Reply:
You could still create local `child` variable to avoid unnecessary code duplication, but it's not that important.

Reply:
done

Reply:
You still call assertNonNull twice on the same element. That's reduntant and calling it once is enough to narrow down type to be non-nullable: `const child = this._assertNotNull(card.children[0], 'card inner div');` here `child` is a non-nullable and you can call methods on it without Typescript complaining that you element might be null